pytm/pytm.py

```diff
@@ -601,6 +604,90 @@ def __str__(self):
         return f"{self.target}: {self.description}\n{self.details}\n{self.severity}"
 
 
+class varColors(var):
+    def __set__(self, instance, value):
+        if not isinstance(value, Iterable):
+            value = [value]
+
+        for i, e in enumerate(value):
+            errors = []
+            if not isinstance(e, str):
+                raise ValueError(
+                    f"expecting a list of str, item number {i} is a {type(e)}"
+                )
+            elif not COLOR_REGEX.match(e):
+                raise ValueError(f"item {i} was not valid hex color code, received {e}")
+
+        # Remove duplicates but preserve order. See https://stackoverflow.com/a/480227 for more
+        # information on why we assign `seen.add` method to a variable (tl;dr it is for speed)
+        seen = set()
+        seen_add = seen.add
+        unique_colors = [e for e in value if not (e in seen or seen_add(e))]
+        super().__set__(instance, unique_colors)
+
+
+class varTMSequenceConfiguration(var):
+    def __set__(self, instance, value):
+        self.data[instance] = value
+
+
+class TMSequenceConfiguration:
+    """Helper class for PlantUML sequence diagrams"""
```

Comment:
Can this be more descriptive? How about 
```suggestion
    """Allows configuring how PlantUML sequence diagrams are drawn"""
```